src/System.Diagnostics.TraceSource/tests/DefaultTraceListenerClassTests.cs

```diff
@@ -45,6 +53,16 @@ public void Fail()
         }
 
         [Fact]
+        public void Fail_WithoutWriteLineOverride()
+        {
+            var listener = new TestDefaultTraceListener();
+            listener.ShouldOverrideWriteLine = false;
+            listener.Fail("FAIL");
+            listener.ShouldOverrideWriteLine = true;
+            Assert.False(listener.Output.Contains("FAIL"));
+        }
+
+        [Fact]
```

Comment:
Do we have any tests that validate the log file that's generated?  Seems like probably not, since it looks like that functionality was only just added.  Can we add some?